dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -88,6 +88,13 @@ def teacher_enrollment_reminder(enrollment)
   end
 
   def exit_survey(workshop, teacher, enrollment)
+    # In case the workshop is reprocessed, do not send duplicate exit surveys.
+    if enrollment.survey_sent_at
+      CDO.warn "Skipping attempt to send a duplicate workshop survey email. Enrollment: #{enrollment.id}"
+      return
+    end
+    enrollment.update!(survey_sent_at: Time.zone.now)
```

Comment:
It's recommended Rails [best practice](http://rails-bestpractices.com/posts/2014/10/22/use-time-zone-now-instead-of-time-now/) to use `Time.zone` because it adjusts to the configured time zone. However all our servers are set to `UTC` so they are equivalent, and we could do either one.